pkg/authenticator/authenticator_factory.go

```diff
@@ -26,10 +26,13 @@ func NewAuthenticatorWithAccessToken(conf config.Configuration, token access_tok
 }
 
 func getAuthenticator(conf config.Configuration, token access_token.AccessToken) (Authenticator, error) {
+	log.Info(log.CAKC075)
```

Comment:
It seems that this log message is not helpful.. at least IMHO... you have a printed log for each case and for the default case.. we will gain the understanding of this function's flow even without this log ..